usr/share/rear/rescue/default/850_save_sysfs_uefi_vars.sh

```diff
@@ -61,7 +61,12 @@ if [[ ! -f ${UEFI_BOOTLOADER} ]]; then
     UEFI_BOOTLOADER=$(find /boot/efi -name "elilo.efi" | tail -1)
 fi
 
-# triple check it
+# in case we have a 64-bit systemd bootloader then we might be lucky with next statements
+if [[ ! -f ${UEFI_BOOTLOADER} ]]; then
+    UEFI_BOOTLOADER=$(find /boot/EFI -name "systemd-bootx64.efi" | tail -1)
```

Comment:
Does Arch uses _/boot/EFI_ mountpoint (in caps)?
So far I've saw only _/boot/efi_ on other distros, and thought it is kind of standard to have _/boot/efi/EFI/..._.

(I've never used Arch before :-( )

V.